lib/name_drop/configuration.rb

```diff
@@ -15,18 +15,4 @@ def initialize
       @access_token = 'NotARealAccessToken'
     end
   end
-
-  class << self
```

Comment:
👍 then this should be good to go!

Sent from my iPhone

> On Sep 12, 2016, at 5:38 PM, Paul Pettengill <notifications@github.com> wrote:
> 
> In lib/name_drop/configuration.rb:
> 
> > @@ -15,18 +15,4 @@ def initialize
> >        @access_token = 'NotARealAccessToken'
> >      end
> >    end
> > -
> > -  class << self
> I see that side of the argument; no need to roll it back.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>